conversant/prompt_chatbot.py

```diff
@@ -297,6 +383,13 @@ def _validate_persona_dict(persona: Dict[str, Any], persona_path: str) -> None:
             persona (Dict[str, Any]): A dictionary containing the persona.
             persona_path: The path from which the persona was loaded.
         """
+
+        # Checks if the parameter does not exceed MAX_GENERATE_TOKENS
+        if persona["client_config"]["max_tokens"] >= MAX_GENERATE_TOKENS:
```

Comment:
Let's add this validation to `configure_client` instead so it is decoupled from persona initialization. This check should still be done even when PromptChatbot is initialized with `__init__`